Jenkinsfile

```diff
@@ -1251,7 +1255,7 @@ pipeline {
                 stage('Functional on Leap 15') {
                     when {
                         beforeAgent true
-                        expression { ! skip_ftest('leap15', env.BRANCH_NAME == 'master') }
+                        expression { ! skip_ftest('leap15', !is_master_build()) }
```

Comment:
Still don't disagree with needlessly reducing testing.  Particularly since it will likely fall on me to have to deduce the reason that something fails on Leap but not on EL7, say.